docs/getting_started/quick_install.md

```diff
@@ -0,0 +1,74 @@
+# Quick install
+
+```{note}
+   These instructions assume familiarity with virtual environments and the
+   [Django web framework](https://www.djangoproject.com/).
+   For more detailed instructions, see [](tutorial).
+   To add Wagtail to an existing Django project, see [](integrating_into_django).
+```
+
+## Dependencies needed for installation
+
+-   [Python 3](https://www.python.org/downloads/).
+-   **libjpeg** and **zlib**, libraries required for Django's **Pillow** library.
+    See Pillow's [platform-specific installation instructions](https://pillow.readthedocs.io/en/stable/installation.html#external-libraries).
+
+## Install Wagtail
```

Comment:
Might be good to run Prettier over these files, while we don't enforce it it's good at picking up inconsistent line breaks like a line under titles. 

Your editor may let you force formatting, or you can enable it temporarily for md files.

```suggestion
## Install Wagtail

```